kuma/dashboards/tests/test_views.py

```diff
@@ -89,7 +93,39 @@ def test_spam_submission_buttons(self):
 
         page = pq(response.content)
         ip_button = page.find('td.dashboard-spam')
-        ok_(len(ip_button) > 0)
+        # User must be logged for spam button column to be visible
+        eq_(len(ip_button), 0)
+
+    def test_submit_akismet_spam_post_required(self):
+        url = reverse('dashboards.submit_akismet_spam', locale='en-US')
+        response = self.client.get(url)
+        eq_(response.status_code, 405, "GET should not be allowed.")
+
+    def test_submit_akismet_spam_valid_response(self):
+        url = reverse('dashboards.submit_akismet_spam', locale='en-US')
+        revision = Revision.objects.first()
+        data = {
+            'revision': revision.pk,
+            'submit': u'spam',
+        }
+        ct = ContentType.objects.get(app_label='attachments',
+                                     model='attachment')
+        p1 = Permission.objects.create(
+            name='Can add Akismet submission',
+            codename='add_revisionakismetsubmission',
+            content_type=ct)
+        admin = User.objects.get(username='admin')
```

Comment:
The permission was already created by the test setup, and can be replaced by:

```p1 = Permission.objects.get(codename='add_revisionakismetsubmission')```

Also, ``admin`` is a superuser, so when you change to ``has_perm`` or similar, the user will have the permission, and no adding of the permission is needed.